### PR TITLE
fix(button): fix fluid style

### PR DESCRIPTION
**Pull request by alcferreira**
affects: @crave/farmblocks-button

ISSUES CLOSED: #109

### CODE REVIEW

File: packages/button/src/styledComponents/Container.js [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
   }
+
+  ${fluidStyle};
 `;
 
+function fluidStyle(props) {
+  return (
+    props.fluid &&
+    css`
+      max-width: 100%;
```
Comment (fczuardi) [inline packages/button/src/styledComponents/Container.js]: is max-width necessary?

File: packages/button/src/styledComponents/Container.js [hunk 2]
```diff
@@ -1,16 +1,27 @@
 // @flow
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 const Container = styled.div`
   position: relative;
-  display: ${props => (props.fluid ? "block" : "inline-block")};
```
Comment (fczuardi) [inline packages/button/src/styledComponents/Container.js]: is the default display value (inline-block) set somewhere else?

Comment (alcferreira) [reply packages/button/src/styledComponents/Container.js]: don't think so... is it necessary anywhere?

File: packages/button/src/styledComponents/Container.js (hunk 1 above)
Comment (alcferreira) [reply packages/button/src/styledComponents/Container.js]: Not really. i'll remove

**Pull request merged**